src/Groups/perm.jl

```diff
@@ -143,6 +143,26 @@ the parent group of $x$ is set to [`symmetric_group`](@ref)$(n)$.
 julia> perm(symmetric_group(6),[2,4,6,1,3,5])
 (1,2,4)(3,6,5)
 ```
+
+Equivalent permutations can be created using [`cperm`](@ref) and [`@perm`](@ref)
+```jldoctest
+julia> x=perm(symmetric_group(8),[2,3,1,5,4,7,8,6])
+(1,2,3)(4,5)(6,7,8)
+
+julia> y=cperm([1,2,3],[4,5],[6,7,8])
+(1,2,3)(4,5)(6,7,8)
+
+julia> x == y
+true
+
+julia> z= @perm (1,2,3)(4,5)(6,7,8)
```

Comment:
```suggestion
julia> z = @perm (1,2,3)(4,5)(6,7,8)
```
Let's be consistent about spacing.

While we don't mention it on https://oscar-system.github.io/Oscar.jl/dev/DeveloperDocumentation/styleguide/ (perhaps we should), I think the majority of code examples puts spaces around `=` and `==` and similar comparison operators. I know that existing examples here are not always consistent, but that's a chance for you (now that you know what at least I prefer) to make things a bit more uniform :-)